core/backend/src/CheckpointManager.ts

```diff
@@ -292,7 +291,7 @@ export class CheckpointManager {
         nativeDb.saveLocalValue("parentChangeSet", JSON.stringify(dbChangeset));
     }
 
-    // SWB
+    // SWB What does context mean here?
     const dbContextGuid = Guid.normalize(nativeDb.queryProjectGuid());

```

Comment:
I believe this is actually out of data with master and has been changed

Reply:
That means iTwin in this context. I think Keith may have fixed this on `master` and changed the "queryProjectGuid". We should check though after a merge.

Reply:
Resolved after a merge.